benchmark/static/static.tsx

```diff
@@ -1,7 +1,9 @@
+#!/usr/bin/env npx ts-node-esm
```

Comment:
I changed the approach here. Instead of using `import-jsx` in an `index.js` file I added a shebang at the top to run the script through `ts-node-esm`, which will use Node loaders to transform the module into standard JS.